src/toggle.jsx

```diff
@@ -216,39 +223,39 @@ const Toggle = React.createClass({
       thumbStyles.marginLeft = '-' + thumbStyles.width;
     }
 
-    let toggleElementStyles = this.mergeStyles(styles.toggleElement, this.props.elementStyle);
+    const toggleElementStyles = Object.assign({},
+      styles.toggleElement,
+      this.props.elementStyle
+    );
 
-    let toggleElement = (
-      <div style={this.prepareStyles(toggleElementStyles)}>
-        <div style={this.prepareStyles(trackStyles)} />
+    const toggleElement = (
+      <div style={prepareStyles(Object.assign({}, toggleElementStyles))}>
+        <div style={prepareStyles(Object.assign({}, trackStyles))} />
         <Paper style={thumbStyles} circle={true} zDepth={1} />
       </div>
     );
 
-    let customRippleStyle = this.mergeStyles({
-      top: -10,
-      left: -10,
-    }, this.props.rippleStyle);
-
-    let rippleColor = this.state.switched ?
-      this.getTheme().thumbOnColor : this.state.muiTheme.textColor;
+    const rippleStyle = Object.assign({},
+      styles.ripple,
+      this.props.rippleStyle
+    );
 
-    let iconStyle = this.mergeStyles(
+    const iconStyle = Object.assign({},
```

Comment:
Actually, the pain points are those ones:
```jsx
    let {
      anchorOrigin,
      className,
      iconButtonElement,
      iconStyle,
      onItemTouchTap,
      onKeyboardFocus,
      onMouseDown,
      onMouseLeave,
      onMouseEnter,
      onMouseUp,
      onTouchTap,
      menuStyle,
      style,
      targetOrigin,
      ...other,
    } = this.props;
````
then one variable is used as a `let`. That's making using `const` at the top impossible.